From dd4ac75db946a6b2c4b577927983fcda464e16f6 Mon Sep 17 00:00:00 2001
From: Volla BuildBot <build@volla.com>
Date: Sat, 28 Jun 2025 08:44:45 +0000
Subject: [PATCH 01/35] Revert "Allow spoofing signingInfo for microG
 Companion/Services"

This reverts commit c60ecdffedf5ec9e05bd92dc022cbb763f49a104.
---
 .../java/com/android/server/pm/ComputerEngine.java  | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/services/core/java/com/android/server/pm/ComputerEngine.java b/services/core/java/com/android/server/pm/ComputerEngine.java
index 6f581306548c..3eb3df412706 100644
--- a/services/core/java/com/android/server/pm/ComputerEngine.java
+++ b/services/core/java/com/android/server/pm/ComputerEngine.java
@@ -170,7 +170,6 @@ import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.PrintWriter;
 import java.nio.charset.StandardCharsets;
-import java.security.cert.CertificateException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collection;
@@ -1591,18 +1590,6 @@ public class ComputerEngine implements Computer {
 
             generateFakeSignature(p).ifPresent(fakeSignature -> {
                 packageInfo.signatures = new Signature[]{fakeSignature};
-                try {
-                    packageInfo.signingInfo = new SigningInfo(
-                            new SigningDetails(
-                                    packageInfo.signatures,
-                                    SigningDetails.SignatureSchemeVersion.SIGNING_BLOCK_V3,
-                                    SigningDetails.toSigningKeys(packageInfo.signatures),
-                                    null
-                            )
-                    );
-                } catch (CertificateException e) {
-                    Slog.e(TAG, "Caught an exception when creating signing keys: ", e);
-                }
             });
 
             return packageInfo;
-- 
2.49.0

